From bc2a622915333d9390a8af9acf87f06008b7fc0d Mon Sep 17 00:00:00 2001
From: Doug Semler <dougsemler@gmail.com>
Date: Tue, 4 May 2010 09:26:27 -0400
Subject: [PATCH 3/8] Fix PR/43538 - CXX_FLAGS_FOR_TARGET set incorrectly on linux

CXX_FLAGS_FOR_TARGET should add flags to the current ones, not
blindly reset them.

ChangeLog:
	* config/mt-gnu: Set CXX_FLAGS_FOR_TARGET as an addition
	  to the current flags.
---
 config/mt-gnu |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/config/mt-gnu b/config/mt-gnu
index 15bf417..5c696f5 100644
--- a/config/mt-gnu
+++ b/config/mt-gnu
@@ -1 +1 @@
-CXXFLAGS_FOR_TARGET = $(CXXFLAGS) -D_GNU_SOURCE
+CXXFLAGS_FOR_TARGET += -D_GNU_SOURCE
-- 
1.7.0.3.254.g4503b

